PxAPI-2.yml

```diff
@@ -811,59 +836,71 @@ components:
       items:
         type: string
       uniqueItems: true
-    # jsonstat-version:
-    #   type: string
-    #   enum:
-    #     - '2.0'
-    #   default: '2.0'
-    # updated:
-    #   oneOf:
-    #     - type: string
-    #       format: date-time
-    #     - type: string
-    #       pattern: "^((19|20)\\d\\d)\\-(0?[1-9]|1[012])\\-(0?[1-9]|[12][0-9]|3[01])$"
     updated:
-        type: string
-        pattern: "^((19|20)\\d\\d)\\-(0?[1-9]|1[012])\\-(0?[1-9]|[12][0-9]|3[01])$"
+      description: See https://json-stat.org/full/#updated
+      type: string
+      pattern: "^((19|20)\\d\\d)\\-(0?[1-9]|1[012])\\-(0?[1-9]|[12][0-9]|3[01])$"
     href:
+      description: Specification on json-stat.org -> [here](https://json-stat.org/full/#href)
       type: string
       format: uri
     label:
+      description: Specification on json-stat.org -> [here](https://json-stat.org/full/#label)
       type: string
     source:
+      description: Specification on json-stat.org -> [here](https://json-stat.org/full/#source)
       type: string
     extension-root:
       description: extension at root level
       type: object
       properties:
         px:
+          description: |
+            Properties corresponds to keywords in the px-file.
+            
+            See [PX file format](https://www.scb.se/en/services/statistical-programs-for-px-files/px-file-format/)
           type: object
           properties:
             infofile:
+              description: Name of a file containing more information for the statistics**
               type: string
             tableid:
+              description: A text that is the identity of the table
               type: string
             decimals:
+              description: The number of decimals in the table cells
               type: integer
             official-statistics:
+              description: Indicates if the data table is included in the official statistics of the organization
               type: boolean
             aggregallowed:
+              description: If the contents of the table cannot be aggregated
               type: boolean
+            refperiod:

```

Comment:
Should be on dimension

Reply:
Removed refperiod for px. It is defined inside extension for dimenison.